exporter/signalfxexporter/exporter.go

```diff
@@ -65,6 +65,7 @@ func New(
 	}
 
 	logger.Info("SignalFx Config", zap.String("ingest_url", options.ingestURL.String()))
+	logger.Info("SignalFx Config", zap.String("api_url", options.apiURL.String()))
```

Comment:
I think we should drop these log statements since for debugging purposes people should send their config files.